createdisk.sh

```diff
@@ -308,6 +308,9 @@ ${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/rhcos-${ostree_hash}/
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/100-crio-bridge.conf'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/200-loopback.conf'
 
+# Remove the journal logs.
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo journalctl --rotate'
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo journalctl -m --vacuum-time=1s'
```

Comment:
is the -m needed?
```
       -m, --merge
           Show entries interleaved from all available journals, including remote ones.
```

Reply:
@cfergeau I just used for safer side to merge any other available journals logs.

Reply:
It says "*Show*" so I don't think this has any effect when vacuuming/rotating.

Reply:
@cfergeau Ack, will remove the `-m` option then.